src/routes/viewRoutes.js

```diff
@@ -0,0 +1,11 @@
+// import express
+const { Router, application } = require("express");
+const { homePage, notePage } = require("../controllers/views/viewControllers");
+
+// create router instance
+const router = Router();
+
+router.get("/notes", notePage);
```

Comment:
Add a `*` route to avoid the app breaking when entering a path other than `/` and `/notes`